File: src/typescript/leetcode/concatenation_of_array/concatenationOfArray.ts
/*
 * @lc app=leetcode id=1929 lang=typescript
 *
 * [1929] Concatenation of Array
 */
// @lc code=start
function getConcatenation(nums: number[]): number[] {
  return nums.concat(nums);
}
// @lc code=end
console.log(getConcatenation([1, 2, 1]));

File: src/python/leetcode/concatenation_of_array/concatenationOfArray.py
#
# @lc app=leetcode id=1929 lang=python3
#
# [1929] Concatenation of Array
#
from typing import List


# @lc code=start
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        return nums * 2


# @lc code=end
s = Solution()
answer = s.getConcatenation([1, 2, 1])
print(answer)
